components/server/src/authorization/authorizer.ts

```diff
@@ -505,6 +507,7 @@ export class Authorizer {
                     optionalSubjectId: relation.subject.object.objectId,
                 },
             },
+            optionalLimit: 0,
```

Comment:
I assume this is put here in order to have spicedb pick the right overload? Maybe worth adding a comment

Reply:
It's an additional parameter that - despite it's name - is required in TypeScript. The usual Go semantics leaking basically. 0 means "no limit" (same semantics as before).
Can add one.

Reply:
ah ok, in that case no need to add a comment. I'm looking through this on GitHub stm. In the IDE it would be clear from the signature.